rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -474,4 +463,9 @@ void disableRecovery() {
             }
         }
     }
+
+    private Collection<LRAData> getLraDataByStatus(Map<URI, Transaction> lrasToFilter, LRAStatus status) {
```

Comment:
Do we need to include `Lra` in the method names, it is clear that these methods are collecting LRAs.
But using it in `lrasToFilter` is good since it explains that the map contains LRAs.

Reply:
+1,  `getDataByStatus` is fine